### ISSUE TITLE
train for less epochs?

### ISSUE THREAD

**Issue by jjallaire** [leonjessen/tensorflow_rstudio_example#1]
In my setup validation accuracy flatlined after ~ 150 epochs:

![rplot](https://user-images.githubusercontent.com/104391/33559266-3e01991c-d8da-11e7-913c-4722a8582b1c.png)

Any objection to training for only 150 epochs?

**Comment by leonjessen** [comment]
150 epochs looks fine - the original 500 was just a "nice number"
